### ISSUE TITLE
[Introspection] `/` is not accepted as character in column name

### ISSUE THREAD

**Issue by janpio** [prisma/prisma2#1078]
```
Error: Schema parsing
error: Unexpected token. Expected one of: field type.
  -->  schema.prisma:67
   | 
66 |   oppID     String?
67 |   R/P       String?
   |    ^ Unexpected token.
68 |   tmID      String?
   | 
```

from

```
model CombinedShutouts {
  date      Int?
  IDgoalie1 Master? @relation("CombinedShutouts_IDgoalie1ToMaster", references: [playerID])
  IDgoalie2 Master? @relation("CombinedShutouts_IDgoalie2ToMaster", references: [playerID])
  month     Int?
  oppID     String?
  R/P       String?
  tmID      String?
  year      Int?
}
```

but

```
CREATE TABLE `CombinedShutouts` (
	`year` INT(11) NULL DEFAULT NULL,
	`month` INT(11) NULL DEFAULT NULL,
	`date` INT(11) NULL DEFAULT NULL,
	`tmID` VARCHAR(255) NULL DEFAULT NULL,
	`oppID` VARCHAR(255) NULL DEFAULT NULL,
	`R/P` VARCHAR(255) NULL DEFAULT NULL,
	`IDgoalie1` VARCHAR(255) NULL DEFAULT NULL,
	`IDgoalie2` VARCHAR(255) NULL DEFAULT NULL,
	INDEX `CombinedShutouts_IDgoalie1` (`IDgoalie1`) USING BTREE,
	INDEX `CombinedShutouts_IDgoalie2` (`IDgoalie2`) USING BTREE,
	CONSTRAINT `CombinedShutouts_ibfk_1` FOREIGN KEY (`IDgoalie1`) REFERENCES `Master` (`playerID`) ON UPDATE CASCADE ON DELETE CASCADE,
	CONSTRAINT `CombinedShutouts_ibfk_2` FOREIGN KEY (`IDgoalie2`) REFERENCES `Master` (`playerID`) ON UPDATE CASCADE ON DELETE CASCADE
)
COLLATE='utf8_general_ci'
ENGINE=InnoDB
;
```

![image](https://user-images.githubusercontent.com/183673/70395912-10aa2980-1a04-11ea-86fe-654e0752cca9.png)

**Comment by janpio** [comment]
Another example:

```
error: Unexpected token. Expected one of: field type.
  -->  schema.prisma:26
   | 
25 |   client_id                       client
26 |   Month/Year                      DateTime
   |        ^ Unexpected token.
27 |   probehla_inzerce_mimo_penezenku String?
   | 
```

https://github.com/prisma/introspection-engine-output/pull/5/files#diff-fdc1fba9ef9c84e028ceae972640def9

**Comment by pantharshit00** [comment]
I can confirm this bug. A temporary workaround here would be to add the `@map` directive to those fields and changing the identifier to a valid name.

**Comment by janpio** [comment]
fixed via #1139
